Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp

```diff
@@ -528,9 +528,15 @@ void test_split()
   std::vector<TriangleMesh> meshes;
 
   PMP::split(tm1, tm2);
+  //try with np
+  typename boost::template property_map<
+  TriangleMesh, CGAL::dynamic_face_property_t<int > >::type
+      pidmap = get(CGAL::dynamic_face_property_t<int>(), tm1);
+  CGAL::Polygon_mesh_processing::connected_components(
+        tm1, pidmap, CGAL::parameters::all_default());
   PMP::split_connected_components(tm1,
                                   meshes,
-                                  params::all_default());
+                                  params::face_patch_map(pidmap));
```

Comment:
Might want to check the result and not just the call.

Reply:
For example put several time the same `off` in a given mesh and check that you have all of them as output + validity + #face/#verteces

Reply:
the result IS tested. Ok, it is expected to be the same result as the default call, but a result nontheless.